### ISSUE TITLE
self-regulate under AdChoices

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay.com#1669]
_**was:** get unblocked by Ghostery_

---

@coderanger reports that https://www.ghostery.com/apps/gittip.

I haven't heard of Ghostery before. What does this mean? What should we do?
##

**Comment by techtonik** [comment]
Closing as there is no `gittip` anymore. When there will be a problem with `gratipay` please open a new ticket.

**Comment by whit537** [comment]
Received in private personal email from []@ghostery.com:

> We're writing to you because you are in our vendor privacy database as a company who provides 3rd party technology on websites. The Digital Advertising Alliance's Self-Regulatory Program encourages such companies to provide direct and functioning opt-outs, however, you do not currently provide one.
>
As one of the approved AdChoices Icon providers under the Program, Ghostery helps companies navigate the self-regulatory "AdChoices" programs in the US, EU and Canada and therefore ask that you contact us with an opt-out script or the contact information of the right person within your company on this issue. 
>
Thank you for your attention to this important matter.

**Comment by whit537** [comment]
I replied:

> Thanks for getting in touch, []. Do you use GitHub? We are an open organization and we have a ticket about this issue:
>
https://github.com/gratipay/gratipay.com/issues/1669
>
You'd be welcome to weigh in there, or you can follow up with support@gratipay.com to help us navigate this process.
> 
Thanks!

**Comment by whit537** [comment]
(Also received on legal@zetaweb.com.)

**Comment by jwheeler66** [comment]
Hello:
My name is Jeff W.  I work at Ghostery.  I was forwarded the email whit537 was referring.  I wanted to reach out and make sure you guys were taken care of.  I see you have some concerns regarding whats being blocked by Ghostery on your pages.  Is their anymore insight you can give me on this?

Jeff

**Comment by whit537** [comment]
Thanks @jwheeler66. In the email quoted above were we asked to "contact [Ghostery] with an opt-out script." What's an "opt-out script" in this context?

**Comment by jwheeler66** [comment]
An opt-out script is a way for a user to opt-out of being tracked if the said company engages in that.  Its a cookie thats usually dropped on a browser telling the company now to track said user.  You can learn more about it here:  http://www.aboutads.info/

**Comment by whit537** [comment]
Thanks @jwheeler66, I've started looking into this, and I've retitled this ticket "self-regulate under AdChoices." We do very little in the way of ads at the moment, and I'm fairly sure we don't track. I'm going to leave this open for down the road when we start doing more of that again.

**Comment by jwheeler66** [comment]
@whit537 great thanks.  Do you still need help with whats being blocked on your site?

**Comment by whit537** [comment]
> Do you still need help with whats being blocked on your site?

If it's obvious to you then we'l be grateful for the input. Otherwise I'm happy to defer this until it's higher priority for us.

**Comment by jwheeler66** [comment]
I should emphasize for the extension Ghostery is NOT just an Ad-blocker but 3rd party tags.  Right now our extension for Gittip blocks anything with gittip.com in the pather.  To me this appears to vauge and will block to much on your site.  I noticed you had a widget, this is something we would normally block.  I found the path for this as (netdna-ssl.net/js/gittip-widget.js).  Is their anything else?

**Comment by whit537** [comment]
Ah, okay. Our widgets are in a bit of a broken state right now. I've added this to our "Finish Team Basics" milestone. Let's circle back when that's further along ...

**Comment by whit537** [comment]
Actually, this really belongs under [Make Widgets 2.0](https://github.com/gratipay/grtp.co/milestone/2), but that's in a different repo and milestones are repo-bound in GitHub (I just mailed support to ask for supra-repo milestones). I've added a stub over there at https://github.com/gratipay/grtp.co/issues/150, and I'm bumping this off the Finish Team Basics milestone.

**Comment by whit537** [comment]
Received in private mail:

> Hi there.
>
>On Feb 1, 2017 the Digital Advertising Association (DAA) along with the Council of Better Business Bureaus (CBBB) will start enforcing their cross device data policies.  [Click Here for Details](http://www.aboutads.info/sites/default/files/DAA_Cross-Device_Guidance-Final.pdf)
To help communicate to consumers which companies are engaging in cross device tracking we will display a Cross Device category with vendors engaged in this activity on our opt-out/opt-in pages.
>
>If your company builds a device graph to better target ad campaigns and marketing activities, please let us know so we can update our company records accordingly. If you can include the name of your company within your reply, that would be greatly appreciated as it will help us navigate our records.
>
>Please make sure you include the name of your company in your response to make it easier for us to locate your records.
>
>Sincerely,
Your Friends at Ghostery

**Comment by whit537** [comment]
Closing in light of our decision to [shut down Gratipay](https://gratipay.news/the-end-cbfba8f50981).

Thank you all for a great run, and I'm sorry it didn't work out! 😞  💃
